common/src/main/java/com/wynntils/features/ui/PlayerInfoFeature.java

```diff
@@ -0,0 +1,252 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.ui;
+
+import static com.wynntils.utils.render.type.HorizontalAlignment.CENTER;
+import static com.wynntils.utils.render.type.HorizontalAlignment.LEFT;
+import static com.wynntils.utils.render.type.VerticalAlignment.MIDDLE;
```

Comment:
Please be consistent and remove the static imports, just use VerticalAlignment.x and HorizontalAlignment.x like you have already in some places